allensdk/brain_observatory/behavior/behavior_ophys_api/behavior_ophys_nwb_api.py

```diff
@@ -170,16 +170,26 @@ def get_segmentation_mask_image(self, image_api=None) -> sitk.Image:
     def get_metadata(self) -> dict:
 
         metadata_nwb_obj = self.nwbfile.lab_meta_data['metadata']
-        data = OphysBehaviorMetaDataSchema(exclude=['experiment_datetime']).dump(metadata_nwb_obj)
+        data = OphysBehaviorMetadataSchema(exclude=['experiment_datetime']).dump(metadata_nwb_obj)
+
+        # Add subject related metadata to behavior ophys metadata
+        nwb_subject = self.nwbfile.subject
```

Comment:
We don't have to worry about compatibility because these data haven't been previously published, correct?

Reply:
Correct!